src/Microsoft.VisualStudio.Services.Agent/ConfigurationStore.cs

```diff
@@ -132,6 +151,14 @@ public bool IsServiceConfigured()
             return serviceConfigured;
         }
 
+        public bool IsAutoLogonConfigured()
+        {
+            Trace.Info("IsAutoLogonConfigured()");
```

Comment:
I just followed the pattern present in the existing methods.

---
In reply to: [122223686](https://github.com/Microsoft/vsts-agent/pull/985#discussion_r122223686) [](ancestors = 122223686)